### DISCUSSION TITLE
三十而立，三十而已

### DISCUSSION THREAD

**Discussion by LeetaoGoooo** [LeetaoGoooo/discussion-blog#239] [随笔]
![30-cake](https://github.com/LeetaoGoooo/discussion-blog/assets/10162120/85492669-8774-401e-8dec-5b61a569a257)

三十而立，我想是每个中国人都能够耳熟能详的一句话，以至于三十岁这年被大家赋予了很多不一样的期许。作为普通人的一员我也免不了俗，三十岁这一年对我来说自然很重要，渴望在三十岁的这年可以完成很多的事情，在三十岁的这年步入人生的新的一个阶段。因此三十岁的生日也自然凸显出很特殊的意义。

特殊的日子做点特殊的事情。在我没有考虑好要做什么的时候，机缘巧合，报名了一个 30 km 徒步的活动，直到活动的前一周，在我对象的提醒下，我才注意到徒步的那一天恰巧是我的生日，30 岁的生日，30 km，每 1km 代表着过去的一年，似乎是个不错的寓意。

![西湖毅行报名截图](https://github.com/LeetaoGoooo/discussion-blog/assets/10162120/7de8810a-44aa-4e0b-8957-63908fac751d)

活动如期而至，除了 30 km 和 30 岁，似乎老天爷也希望我这天可以难以忘怀一些，杭州的这天的温度也来到了 30 多度。1 km 代表一岁，在山路中穿梭，偶尔驻足的时候，我在想这一年有没有发生什么让我印象深刻的事情。

1km,2km,..., 17 km, 好像并没有，直到来到了 18 km，是的，18 岁的那一年，恰逢成年那年，经历过酷夏的高考，从高中生变成了大学生，似乎人生来到了一个新的阶段。

![18km](https://github.com/LeetaoGoooo/discussion-blog/assets/10162120/6921abef-c30d-4841-b7b4-dca8211f60ec)

19 km,20 km,... 23 km, 23 岁的那一年，毕业工作的第一年，彻底地从学生完成了到打工人的转变，又是一个新的阶段的开始，一晃眼，来到了 24 年，工作已经第八个年头了，经历过不少事情，周转了几个城市，换了几份工作，也结识了不少有趣的小伙伴。工作之余，偶尔也会思考所谓的人生的意义，可惜的是，到现在我还没有思考明白，明不明白不要紧，迷茫地的时候做一些总归是对的事情是肯定没有错的：认真地生活，好好锻炼身体，坚持读书，没事多出去走走。

![23km](https://github.com/LeetaoGoooo/discussion-blog/assets/10162120/a8dcd969-7779-4fb0-9be7-e570f9993fd6)

30 km 的路上像人生之路的缩影，会有上坡，上坡的时候我们会累，但是我们切切实实地正在往上走着；有下坡，坡可能会很陡，脚底可能会打滑，我们可能会摔倒，但是并没有关系，我们可以拍拍身上的灰尘，然后再爬起来，没有永远下坡的路，坚持一下，下坡到底之后等待我们的不是平路就是上坡了，人生的路上会有很多人从我们身边擦身而过，我们会走到很多人的前面，我们也会被很多人超过，会有人在我们身边短暂停留而后离去，也会有人一直陪着我们一直走到了终点，我们不能强求别人去做什么，我们唯一可以做的就是要求自己尽力去做到什么。

30 岁生日是什么感觉，**30而已**。

![自拍](https://github.com/LeetaoGoooo/discussion-blog/assets/10162120/6c767eab-f94b-4f38-81d2-af79efefbfb9)

**Comment by lg2it** [comment]
生日快乐

**Reply by LeetaoGoooo** [reply]
谢谢～😄

**Comment by Dieinwarm** [comment]
你这行动力太强了，体能也强啊，祝福晚了两天，生日快乐呀~~

**Reply by LeetaoGoooo** [reply]
> 你这行动力太强了，体能也强啊，祝福晚了两天，生日快乐呀~~

哈哈 你也可以的，谢谢啦～
